### PR TITLE
Fixed typos in the documentation

### LINKED ISSUE #1683: [BUG] Typo in docs
**Describe the bug**
There are some typos in the documentation.

**How to reproduce**

**Expected behavior**

**Actual behavior**
What actual happened.

**Code examples**
Sample code (or your configuration) to help us reproduce the bug faster or to understand your use case.
```
//Write code here
```
Or add code snippets (Code sandbox)

**Screenshots**

**Devices (please complete the following information):**
 - Device: [e.g. iPhone6, Desktop]
 - Browser: [e.g chrome, safari][version]

**Additional context**
Anything you would like to add but was not sure which section it fits. For example: tech stack used (Nuxt, Vue, Gridsome, React, etc)

### PR DESCRIPTION
**Pull request by ymaheshwari1**
# Related issue

Closes #1683 

# Scope of work

# Screenshots of visual changes

# Checklist

- [x] No commented blocks of code left
- [ ] Run tests and docs
- [ ] Self code-reviewed
- [ ] Changes documented 

If applicable:

- [x] I followed [composition rules](https://docs.storefrontui.io/?path=/story/introduction-contributing-guide-code-guidelines--page) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

**Review by justyna-13 — APPROVED** [review]

File: packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx [hunk 1]
```diff
@@ -106,7 +106,7 @@ Anything for a better StorefrontUI and bulding the community is considered contr
 
 ### Contribute Code
 
-We have a lot of features that need good helping hand, as always. However, there are coding rules you need to follow, for maintaining readability and for us to process your pull request as quickly as possible. Please read [our code contributing guide](?path=/story/introduction-contributing-guide-code-guidelines--page).
```
Comment (justyna-13) [inline packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx:109]: `?path=/story/` should be here :)

**Review by justyna-13 — COMMENTED** [review]
Good job @ymaheshwari1 :+1: please see my comment :slightly_smiling_face:

File: packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx (hunk 1 above)
Comment (ymaheshwari1) [reply packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx:109]: Sure, I will fix that

**Review by Baroshem — APPROVED** [review]
Looks good to me. Well done!

Please rebase your branch to be up to date with our `develop` branch so we could merge it :)

**Comment by ymaheshwari1** [comment]
> @ymaheshwari1
> 
> Looks good to me. Well done!
> 
> Please rebase your branch to be up to date with our `develop` branch so we could merge it :)

Sure I will update it.

File: packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx (hunk 1 above)
Comment (ymaheshwari1) [reply packages/vue/src/stories/introduction/3. contributing/1. contribute.stories.mdx]: @justyna-13 I have updated the link.

**Review by Baroshem — APPROVED** [review]

**Comment by Baroshem** [comment]
@justyna-13 please check once again ;)

**Review by justyna-13 — APPROVED** [review]
:+1:

**Pull request merged**